### PR TITLE
Fixes external server tests and change other references.

### PR DESCRIPTION
**Pull request by roshkhatri**
Fixes external server tests and change other references.

### CODE REVIEW

**Review by PingXie — COMMENTED** [review]
Thanks! Changes LGTM but I think @madolson should have a look and the final say.

**Review by zuiderkwast — APPROVED** [review]
LGTM but we'll have to change all occurrences of placeholderkv again once we have a name for the project. Depending on how long that takes, we could wait for that before merging this.

**Comment by madolson** [comment]
@zuiderkwast eh, there are some non-overlapping changes here as well (like the flags changes). We'll have to do a lot of scrubbing, doing a find and replace on placeholderkv is much faster than redis.

File: .github/workflows/daily.yml
Comment (zuiderkwast) [inline .github/workflows/daily.yml:51]: Let's add this renamed variable (not the CI but the Makefile change) to release notes. It's an interface change.

**Review by zuiderkwast — APPROVED** [review]

**Pull request merged**
